apps/web/package.json

```diff
@@ -17,6 +17,12 @@
     "next": "13.0.2",
     "react": "18.2.0",
     "react-dom": "18.2.0",
+    "tailwindcss": "^3.2.4",
```

Comment:
This is related to the explanation in my PR

Reply:
Awesome! Looks great!